File: app/src/main/java/br/com/santander/testeandroid/ui/contact/domain/models/Cell.java
package br.com.santander.testeandroid.ui.contact.domain.models;

import com.google.gson.annotations.SerializedName;

public class Cell {
    private int id;
    private Type type;
    private String message;
    @SerializedName("typefield")
    private TypeField typeField;
    private boolean hidden;
    private int topSpacing;
    private int show;
    private boolean required;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Type getType() {
        return type;
    }

    public void setType(Type type) {
        this.type = type;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public TypeField getTypeField() {
        return typeField;
    }

    public void setTypeField(TypeField typeField) {
        this.typeField = typeField;
    }

    public boolean isHidden() {
        return hidden;
    }

    public void setHidden(boolean hidden) {
        this.hidden = hidden;
    }

    public int getTopSpacing() {
        return topSpacing;
    }

    public void setTopSpacing(int topSpacing) {
        this.topSpacing = topSpacing;
    }

    public int getShow() {
        return show;
    }

    public void setShow(int show) {
        this.show = show;
    }

    public boolean isRequired() {
        return required;
    }

    public void setRequired(boolean required) {
        this.required = required;
    }
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/Cell.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Cell: Serializable {

    @SerializedName("id")
    var id: Int? = null
    @SerializedName("type")
    var type: Int? = null
    @SerializedName("message")
    var message: String? = null
    @SerializedName("typefield")
    var typeField: String? = null
    @SerializedName("hidden")
    var hidden: Boolean? = null
    @SerializedName("topSpacing")
    var topSpacing: Double? = null
    @SerializedName("show")
    var show: Int? = null
    @SerializedName("required")
    var required: Boolean? = null

    override fun toString(): String {
        return "Cell(id=$id, type=$type, typeField=$typeField, " +
                "hidden=$hidden, topSpacing=$topSpacing, show=$show, required=$required)"
    }

}